src/svgs/index.tsx

```diff
@@ -1,43 +1,69 @@
-export * from "./ArtsyLogoIcons"
+export * from "./AddCircleFillIcon"
+export * from "./AddCircleIcon"
+export * from "./AddIcon"
+export * from "./AlertIcon"
+export * from "./ArrowDownCircleIcon"
+export * from "./ArrowDownIcon"
+export * from "./ArrowLeftCircleIcon"
+export * from "./ArrowLeftIcon"
+export * from "./ArrowRightCircleIcon"
+export * from "./ArrowRightIcon"
+export * from "./ArrowUpCircleIcon"
+export * from "./ArrowUpIcon"
+export * from "./ArtsyLogoBlackIcon"
+export * from "./ArtsyLogoWhiteIcon"
+export * from "./ArtsyMarkBlackIcon"
+export * from "./ArtsyMarkWhiteIcon"
+export * from "./ArtworkIcon"
 export * from "./AuctionIcon"
+export * from "./BellFillIcon"
 export * from "./BellIcon"
 export * from "./BlueChipIcon"
-export * from "./BookIcon"
+export * from "./CheckCircleFillIcon"
+export * from "./CheckCircleIcon"
 export * from "./CheckIcon"
 export * from "./ChevronIcon"
-export * from "./ChevronIcon"
-export * from "./CircleBlackCheckIcon"
-export * from "./CircleWhiteCheckIcon"
-export * from "./ClosedEyeIcon"
+export * from "./CloseCircleIcon"
 export * from "./CloseIcon"
+export * from "./CollapseIcon"
 export * from "./CreditCardIcon"
+export * from "./DocumentIcon"
 export * from "./DownloadIcon"
 export * from "./EditIcon"
+export * from "./EnterIcon"
+export * from "./EnvelopeIcon"
+export * from "./EstablishedIcon"
+export * from "./ExpandIcon"
+export * from "./EyeClosedIcon"
+export * from "./EyeOpenedIcon"
 export * from "./FacebookIcon"
 export * from "./FairIcon"
 export * from "./FilterIcon"
 export * from "./GenomeIcon"
-export * from "./GroupIcon"
+export * from "./GraphIcon"
+export * from "./HeartFillIcon"
 export * from "./HeartIcon"
-export * from "./HelpIcon"
-export * from "./Icon"
+export * from "./HomeIcon"
```

Comment:
Can we add `// TODO's` around the icons that have been aliased so that there's a quick reference list?